templates/terraform/processing-project/buckets.tf

```diff
@@ -5,6 +5,17 @@ resource google_storage_bucket artifact_bucket {
   location = "US"
 }
 
+resource google_storage_bucket_iam_member artifact_bucket_uploader_iam {
+  provider = google.target
+  bucket = google_storage_bucket.artifact_bucket.name
+  # When the storage.admin role is applied to an individual bucket,
+  # the control applies only to the specified bucket and objects within
+  # the bucket: https://cloud.google.com/storage/docs/access-control/iam-roles
+  role = "roles/storage.legacyBucketWriter"
```

Comment:
Why `legacyBucketWriter` instead of one of the newer object roles?

Reply:
Not sure if the newer ones have just object-specific permission: https://cloud.google.com/storage/docs/access-control/iam-roles#legacy-roles, maybe roles/storage.objectCreator is what we're looking for?